projects/demo/src/modules/components/combo-box/examples/7/index.html

```diff
@@ -0,0 +1,28 @@
+<form class="b-form">
+    <tui-combo-box
+        class="b-form"
+        [ngModelOptions]="{standalone: true}"
```

Comment:
You can just remove wrapping `form` tag, it's unnecessary here

Reply:
fixed